cas-server-webapp/src/main/webapp/WEB-INF/cas.properties

```diff
@@ -691,3 +691,8 @@ accept.authn.users=casuser::Mellon
 # cas.pac4j.oidc.customParamValue1=
 # cas.pac4j.oidc.customParamKey2=
 # cas.pac4j.oidc.customParamValue2=
+
+##
+# CAS OAuth server support: keys are mandatory for JWT support
+cas.oauth.jwt.signingSecret=
```

Comment:
I meant why not `# cas.oauth.jwt.signingSecret=` ? Most settings I think should be commented out unless needed. Otherwise they would show up in the UI under the properties report and might be misleading.